File: miniblog_front/src/server/dao/Card.js
import Vue from "vue"

const PADS_REST = "/miniblog/data/card";

function Card() {
    this.get = function (id) {
        return Vue.http.get(PADS_REST + "/" + id);
    };

    this.delete = function (id) {
        return Vue.http.delete(PADS_REST + "/" + id);
    };

    this.put = function (pad) {
        return Vue.http.put(PADS_REST+ "/" + pad.id, pad);
    };

}

export default Card;

File: miniblog_back/src/main/java/red/medusa/miniblog/web/pad/bean/Card.java
package red.medusa.miniblog.web.pad.bean;

import com.fasterxml.jackson.annotation.JsonFormat;

import java.io.Serializable;
import java.util.Date;

public class Card implements Serializable {
    private String id;
    private String content;
    @JsonFormat(timezone = "GMT+8", pattern = "yyyy-MM-dd")
    private Date updateTime;
    private Boolean isChecked;
    private Boolean editable;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public Boolean getIsChecked() {
        return isChecked;
    }

    public void setIsChecked(Boolean checked) {
        isChecked = checked;
    }

    public Date getUpdateTime() {
        return updateTime;
    }

    public void setUpdateTime(Date updateTime) {
        this.updateTime = updateTime;
    }

    public Boolean getEditable() {
        return editable;
    }

    public void setEditable(Boolean editable) {
        this.editable = editable;
    }

    @Override
    public String toString() {
        return "Card{" +
                "id='" + id + '\'' +
                ", content='" + content + '\'' +
                ", updateTime=" + updateTime +
                ", isChecked=" + isChecked +
                ", editable=" + editable +
                '}';
    }
}
